xarray/tests/test_plot.py

```diff
@@ -1828,6 +1830,102 @@ def test_datetime_line_plot(self):
         self.darray.plot.line()
 
 
+@requires_nc_time_axis
+@requires_cftime
+class TestCFDatetimePlot(PlotTestCase):
+    @pytest.fixture(autouse=True)
+    def setUp(self):
+        '''
+        Create a DataArray with a time-axis that contains cftime.datetime
+        objects.
+        '''
+        # case for 1d array
+        month = np.arange(1, 13, 1)
+        data = np.sin(2 * np.pi * month / 12.0)
+        time = xr.cftime_range(start='2017',
+                               periods=12,
+                               freq='1M',
+                               calendar='noleap')
+        darray = DataArray(data, dims=['time'])
+        darray.coords['time'] = time
+
+        self.darray = darray
+
+        # case for 2d arrays
+        data_2d = np.random.rand(4, 12)
+        darray_2d = DataArray(data_2d, dims=['x', 'time'])
+        darray_2d.coords['time'] = time
+
+        self.darray_2d = darray_2d
+
+    def test_cfdatetime_line_plot(self):
+        # test if line plot raises no Exception
+        self.darray.plot.line()
+
+    def test_cfdatetime_pcolormesh_plot(self):
+        # test if line plot raises no Exception
```

Comment:
In general I'm not sure if these comments are really necessary here (though I realize they were copied from the `TestDatetimePlot` test class I pointed you to as an example).  If you choose to keep them, at least update them for the different test cases (e.g. this should be `pcolormesh` rather than `line`).

Reply:
I deleted them.